src/mqt/bench/calibration_files/iqm_adonis_calibration.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "iqm_adonis",
+  "num_qubits": 5,
+  "basis_gates": ["r", "cz", "measure", "barrier"],
```

Comment:
Definitions for these gates can be found in the Cirq documentation:
- https://quantumai.google/reference/python/cirq/PhasedXPowGate
- https://quantumai.google/reference/python/cirq/XPowGate
- https://quantumai.google/reference/python/cirq/YPowGate

Reply:
Well, that is curious. I got the structure for the calibration data from [this](https://github.com/iqm-finland/qiskit-on-iqm/tree/main/src/iqm/qiskit_iqm/fake_backends) source. Also, I have rechecked with the LRZ lab team, and they have confirmed the native gate set is, in fact, "r" and "cz".

Reply:
I doublechecked now with the hardware guys and they reassured me "r" and "cz" are the correct native gates.
This is consistent with the [iqm-client](https://iqm-finland.github.io/iqm-client/api/iqm.iqm_client.iqm_client.html#module-iqm.iqm_client.iqm_client) and [qiskit-on-iqm](https://github.com/iqm-finland/qiskit-on-iqm/tree/main/src/iqm/qiskit_iqm). I also found [this](https://github.com/iqm-finland/cirq-on-iqm/blob/main/src/iqm/cirq_iqm/iqm_operation_mapping.py) mapping for the cirq-on-iqm.

Reply:
Thanks for double checking! I suspected as much. The R gate is just the phased x-pow gate and the other two gates are special cases where certain angles are set to some fixed values. So the R gate can represent all the possible gates.
What I would assume though (could be wrong), is that the simpler gates typically have better fidelities and are faster. So, in the future, it might make sense to differentiate between them, just because they are calibrated differently.
For now, the R gate should be fine and the appropriate definition for TKET is given by the special paramterization of the U gate.